powertools-parameters/powertools-parameters-appconfig/pom.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>software.amazon.lambda</groupId>
+        <artifactId>powertools-parent</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
```

Comment:
Then we have one of these per-provider, stored in the `powertools-parameters` subdirectory, pulled into the entire repo parent POM.

@jeromevdl what do you think - should we nest this at all, or just have all the modules at the top directory? I am now leaning towards the latter - some stuff (like the relative path to `spotbugs-exclude.xml`) is easier when all of the deployable modules are one level deep.

Reply:
I agree it's probably easier to keep it on root, but I think we start to have too many things at the root level. Would be cleaner to have it nested. And I should probably do the same for logging. Have one logging and then the log4j / logback implementations nested.

Reply:
I'll leave it like it is, then